### DISCUSSION TITLE
Case logs customization

### DISCUSSION THREAD

**Discussion by dgmyr** [Security-Onion-Solutions/securityonion#10313] [Unsupported Versions]
It's possible to add existing in so_kind: case logs fields to another so_kinds in so-case and so-casehistory index. Example add so_case.title field to related, comment, artifact so_kinds?

**Comment by weslambert** [comment]
I'm not sure what you asking. It is somewhat unclear. Could you please try rephrasing your question?

**Reply by dgmyr** [reply]
In the logs of indexes so-case* and so-casehistory* there is 'so_kind' field that corresponds to a kind of activity perfomed on cases. 
I would like to know if it's possible to include fields present in logs with 'so_kind: case' (these logs corresponds to an activity on cases, like creation and update) to another kind, like logs that corresponds to related events ('so_kind: related') in order to improve searching in kibana dashboards. So in practice i would like to add, for example the 'so_case.title' field to all logs of of indexes so-case* and so-casehistory* , not only to the logs with  'so_kind: case'.
